NEWS.md

```diff
@@ -2,10 +2,21 @@
 
 ## Unreleased
 
+### Performance
+- Pipeline store writes #3084 #3177
+
+### New DB table for dynamic data sources
+
+For new subgraph deployments, dynamic data sources will be recorded under the `sgd*.data_sources$`
+table, rather than `subgraphs.ethereum_contract_data_source`. As a consequence
```

Comment:
That should be `subgraphs.dynamic_ethereum_contract_data_source`

Reply:
Thanks!